cfgov/jinja2/v1/_includes/organisms/full-width-text.html

```diff
@@ -7,8 +7,9 @@
 
    Description:
 
+   See [GHE]/CFPB/hubcap/wiki/Full-Width-Copy
```

Comment:
How about we rename this wiki page to "Full Width Text" and update this link, since that's how it's referred to everywhere else? (Including in the text of said wiki page.)

Reply:
All good by me! @anselmbradford